sdk/go/audit.sdk.gen.go

```diff
@@ -0,0 +1,11 @@
+// Code generated by protoc-gen-go-flipt-sdk. DO NOT EDIT.
+
```

Comment:
@erka can we delete this and all the generated files under `sdk/go` around auditing? i don't think they are necessary

Reply:
Spot on. I modified protoc-gen-go-flipt-sdk to skip generating code for proto files that don't contain grpc services